### ISSUE TITLE
Responsive Website on mobile phone

### ISSUE THREAD

**Issue by LeonStra** [ls1intum/Artemis#8845]
### Describe the bug

You can't use artemis on mobile devices, because the text doesn't end on the edge of the display and you are not able to scoll horizontal.

### To Reproduce

1. Go to https://artemis.cit.tum.de/
2. Click on any course and any task

### Expected behavior

See the whole task

### Which version of Artemis are you seeing the problem on?

7.2.2

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

**Comment by krusche** [comment]
We have dedicated mobile apps for Artemis, but also want to ensure that at least some parts of the web app work on mobile as well. Can you please be more specific and provide screenshots?

**Comment by LeonStra** [comment]
Here is an example. I cant scroll to the right, but I could view it in Desktop Mode. There is an mobile App? I couldn't find one at the Google play Store. Is there an APK?

[Image]



Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by krusche** [comment]
I agree that a responsive website is desirable. However, with a complex website such as Artemis, this is not easily done. I suggest that we focus on smaller issues with concrete URLs where the issue takes place and avoid a generic issue where developers do not really know how to approach it

**Comment by krusche** [comment]
Feel free to create specific issues (with a specific title) where the website is not responsive on mobile. We will try our best to address them
